h/static/scripts/group-forms/components/CreateGroupForm.tsx

```diff
@@ -3,27 +3,52 @@ import { Button, Input, Textarea } from '@hypothesis/frontend-shared';
 export default function CreateGroupForm() {
   return (
     <>
-      <h1>Create a new private group</h1>
+      <h1 class="mt-[55px] mb-[30px] text-[#3f3f3f] text-[19px] leading-[15px]">
```

Comment:
Note the `font-size: 19px; line-height: 15px;`. This is necessary if we want the vertical position of the heading to exactly match the legacy form. But having a `line-height` that's smaller than the `font-size` is non-sensical and looks accidental in the legacy code (the `line-height` is inherited from CSS applied to the `<body>` element).